components/autocomplete/events.md

```diff
@@ -227,6 +228,62 @@ When using `OnRead`, make sure to set `TItem` and `TValue`.
 }
 ````
 
+## OnItemRender
+
+The `OnItemRender` event fires when each item in the collection passed to the `Data` parameter of the AutoComplete renders. 
+
+The event handler receives as an argument an `AutoCompleteItemRenderEventArgs<TItem>` object that contains:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Property | Description                              |
+|----------|------------------------------------------|
+| `Item`   | The current item that renders in the AutoComplete. |
+| `Class`  | The custom CSS class that will be added to the item.     |
+
+````CSHTML
+@* Disable an item in the AutoComplete *@
+
+<style>
+    .disabled-item{
+        pointer-events: none;
+        color: gray;
+    }
+</style>
+
+<TelerikAutoComplete Data="@Suggestions"
+                     OnItemRender="@OnItemRenderHandler"
+                     ValueField="@( nameof(SuggestionsModel.Suggestion) )"
+                     @bind-Value="@TheValue" />
+
+@code {
+    string TheValue { get; set; }
```

Comment:
`TheValue` is not according to our style guide. How do you distinguish properties if you have two components on the page?

```suggestion
    private string TheValue { get; set; }
```